ReportingServicesTools/Functions/CatalogItems/Get-RsSubscription.ps1

```diff
@@ -52,7 +57,11 @@ function Get-RsSubscription
         [System.Management.Automation.PSCredential]
         $Credential,
         
-        $Proxy
+        $Proxy,
+
+        [ValidateSet('2005','2010')]
+        [string]
+        $APIVersion = '2010'
```

Comment:
Please rename this to ApiVersion.

Reply:
Could you also please move ApiVersion after ReportServerUri?

Reply:
Any reason why 2006 is missing from this list?

Reply:
I wasn't sure whether it would work against the 2006 API, as it seemed to be a completely different set of methods (I read that the 2006 API is geared around working with Sharepoint). I'll add it though and test.